RetailCoder.VBE/UI/Command/AddTestMethodCommand.cs

```diff
@@ -47,7 +47,18 @@ public AddTestMethodCommand(IVBE vbe, RubberduckParserState state) : base(LogMan
 
         protected override bool EvaluateCanExecute(object parameter)
         {
-            if (_state.Status != ParserState.Ready || _vbe.ActiveCodePane == null) { return false; }
+            if (_state.Status != ParserState.Ready)
```

Comment:
Also beyond the PR's status but I observe several, if not all `EvaluateCanExecute` has this check for the `ParserState.Ready`. This makes me wonder if the base should take on implementing it to allow derived classes to focus on their thing.

Reply:
Would have to be a templated method then.